src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Models/ChangeOfPartyRequest/CreateCohort/WhenCohortIsCreated.cs

```diff
@@ -139,7 +139,7 @@ public void Then_ChangeOfPartyRequestId_Is_Correct(ChangeOfPartyRequestType requ
 
         [TestCase(ChangeOfPartyRequestType.ChangeEmployer)]
         [TestCase(ChangeOfPartyRequestType.ChangeProvider)]
-        public void Then_AssignedToOtherPartyEvent_Is_Emitted(ChangeOfPartyRequestType requestType)
+        public void And_RequestTypeIsChangeOfEmployer_Then_AssignedToOtherPartyEvent_Is_Emitted(ChangeOfPartyRequestType requestType)
```

Comment:
The method name is changed to requestType is change of employer but both the the request types are passed in.